docs/docs/get-started/get-started.md

```diff
@@ -0,0 +1,82 @@
+---
+id: get-started
+title: Kurtosis Documentation
+slug: '/'
+sidebar_position: 1
+hide_table_of_contents: true
+---
+
+[Kurtosis](https://github.com/kurtosis-tech/kurtosis) is a tool for packaging and launching environments of containerized services where you want them, the way you want them, with one liners.
```

Comment:
```suggestion
Our mission is to make building distributed systems as easy as building a single server application. The first problem we aim to solve on our journey to achieve this mission is to bridge the author-consumer divide for environment definitions. For this reason, we built Kurtosis.

[Kurtosis](https://github.com/kurtosis-tech/kurtosis) is a tool for packaging and launching environments of containerized services where you want them, the way you want them, with one liners. 
```

Reply:
not going to commit this one - don't want to tell new users coming into the first page of our docs about our mission, I just want to tell them how we're helping them right now. later as they dig into the product and understand it more, we can give context that helps understand the roadmap, where its going, and why its going there